### ISSUE TITLE
Webpack bundler and mupX

### ISSUE THREAD

**Issue by mixersoft** [Urigo/IonicCLI-Meteor-WhatsApp#2]
I'm not familiar with bundling and 'webpack' but I'd like to know how this configuration would be deployed to a hosted server. Is it compatible with 'mupX' ?

**Comment by DAB0mB** [comment]
`meteor` uses it's own bundling system, so there might be conflicts with the two, I can't really tell if it will work with `mupX` properly. In general, before deploying any server you should bundle your modules beforehand, so the client can receive the bundled version of your app.

**Comment by mixersoft** [comment]
I tried to run the demo from the github repo and got an error on `gulp` (see the issue I opened). However, I wanted to see if the Meteor server code is run from from the same http server as the client App, i.e. `ionic serve -> http://localhost:8100` 

Right now, I have an `angular-meter + ionic CLI` deployment where I put `meteor` in the `/path/to/ionic/project/meteor` folder. I'm serving Meteor from a different server using the following:

```
cd /path/to/ionic/project/meteor
meteor run --port 3333 --settings ./settings.json & 
```

It works, but I have to do a lot of crazy stuff to get the `meteor-client-side` code to work from `ionic` - especially since some of the packages, like `accounts-facebook` also expect the server-side code to run from the same `ROOT_URL` or have client-side dependencies on `Meteor.settings.public`

see: https://github.com/mixersoft/accounts-facebook-client-side/blob/master/README.md

However, with the Meteor server code neatly stored in a subfolder, it is easy to deploy via `mupx` - especially since I tried an `rsync` deploy of the Meteor server and it wouldn't work inside an `ec2 micro` instance. I don't know enough about `mpux` or the meteor bundling system to understand if this would still be the case when I have `client` and `server` side code mixed together via the `webpack` bundling system of the demo. 

Would it make my life easier to convert the project build to `webpack`? Can I still deploy the whole thing to an `ec2 micro` instance?
